### PR TITLE
COR-732: fix form labels

### PR DESCRIPTION
**Pull request by internetti**
I took some liberties where it felt to make more sense

### CODE REVIEW

File: internal/views/individual_form.go [hunk 1]
```diff
@@ -323,152 +323,152 @@ func (f *IndividualForm) buildSex() error {
...
+		DisplayName: "Date of Birth",
 		MinValue:    "1900-01-01",
 	}, f.personalInfoSection, f.individual.BirthDate)
 }
 
 func (f *IndividualForm) buildAge() error {
 	return buildField(&forms.NumberInputField{
 		Name:        "age",
-		DisplayName: "Age",
+		DisplayName: "Age of individual",
```
Comment (neb42) [inline internal/views/individual_form.go:334]: participant

File: internal/views/individual_form.go [hunk 2]
```diff
@@ -291,14 +291,14 @@ func (f *IndividualForm) buildMiddleName() error {
 func (f *IndividualForm) buildLastName() error {
 	return buildField(&forms.TextInputField{
 		Name:        "lastName",
-		DisplayName: "Last Name",
+		DisplayName: "Surname",
```
Comment (neb42) [inline internal/views/individual_form.go:294]: Was this requested? I feel last name is less prone to localisation differences

**Review by neb42 — APPROVED** [review]
Unsure which ones you did yourself, but it may be worth running by Fix before merging

File: internal/views/individual_form.go (hunk 2 above)
Comment (internetti) [reply internal/views/individual_form.go:294]: Yeah, wasn't sure about this one, the table said Surname.

**Pull request merged**